scene/main/window.cpp

```diff
@@ -506,7 +506,9 @@ void Window::request_attention() {
 	}
 }
 
+#ifndef DISABLE_DEPRECATED
 void Window::move_to_foreground() {
+	WARN_DEPRECATED_MSG(R"*(The "move_to_foreground()" method is deprecated, use "grab_focus()" instead.)*");
 	ERR_MAIN_THREAD_GUARD;
 	if (embedder) {
 		embedder->_sub_window_grab_focus(this);
```

Comment:
If we're going with the deprecation, I'd suggest you still also amend the commit to replace the code by just `grab_focus();` (plus the deprecation warning you added already).